packages/docs/src/stories/templates/header.stories.ts

```diff
@@ -113,34 +129,34 @@ export const SampleA02 = {
       <div class="flex justify-between items-center">
         <!-- top-left-area start !-->
         <a class="flex flex-shrink" href="#">
-          <img class="h-8 md:h-12 lg:h-14" src="images/logo-unioninvestment-lg.svg" alt="Logo" />
+          <img class="h-8 md:h-12 lg:h-14" src="images/logo-unioninvestment-lg.svg" alt="Union Investment Homepage" />
         </a>
         <!-- top-left-area end !-->
         <!-- top-right-area start !-->
         <div class="flex lg:hidden">
-          <sd-navigation-item onclick="openDrawerSampleA2()">
-            <sd-icon name="system/menu" class="text-xl  -my-[1.5px] -mx-1"></sd-icon>
+          <sd-navigation-item id="open-menu-sample-a-02">
+            <sd-icon name="system/menu" label="Open menu" class="text-xl  -my-[1.5px] -mx-1"></sd-icon>
           </sd-navigation-item>
         </div>
         <div class="-mr-4 lg:flex hidden items-center">
           <sd-navigation-item href="javascript:void(0)">
             <div class="w-4 h-4 flex items-center">
-              <sd-icon name="system/filter-empty" class="text-xl absolute -ml-1"></sd-icon>
+              <sd-icon name="system/filter-empty" label="Filter empty" class="text-xl absolute -ml-1"></sd-icon>
             </div>
           </sd-navigation-item>
           <sd-navigation-item href="javascript:void(0)">
             <div class="w-4 h-4 flex items-center">
-              <sd-icon name="system/globe" class="text-xl absolute -ml-1"></sd-icon>
+              <sd-icon name="system/globe" label="Globe" class="text-xl absolute -ml-1"></sd-icon>
```

Comment:
```suggestion
              <sd-icon name="system/globe" label="Languages" class="text-xl absolute -ml-1"></sd-icon>
```

Or maybe language switcher.

Reply:
Updated